<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1H17STS" failures="1" tests="2" name="Surefire test" time="0.067" errors="0" timestamp="2023-08-03T13:49:32 IST">
  <testcase classname="com.module1.mavencore.CalculatorTest" name="subTest" time="0.005"/>
  <testcase classname="com.module1.mavencore.CalculatorTest" name="sumTest" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [200] but found [30]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [30]
at com.module1.mavencore.CalculatorTest.sumTest(CalculatorTest.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- sumTest -->
</testsuite> <!-- Surefire test -->
